packages/components/src/templates/next/components/internal/Wizgov/WizgovWidgetClient.tsx

```diff
@@ -0,0 +1,45 @@
+"use client"
+
+import { useEffect } from "react"
+
+import type { WizgovWidgetClientProps } from "~/interfaces"
+
+const reloadWizgovScript = (
+  scriptUrl: WizgovWidgetClientProps["scriptUrl"],
+) => {
+  const scriptId = "isomer-wizgov-script"
+
+  const existingScriptTag = document.getElementById(scriptId)
+  if (existingScriptTag) {
+    existingScriptTag.remove()
+  }
+
+  const scriptTag = document.createElement("script")
+  scriptTag.id = scriptId
+  scriptTag.async = true
+  scriptTag.type = "text/javascript"
+  scriptTag.src = scriptUrl
+  scriptTag.referrerPolicy = "origin"
+  document.body.appendChild(scriptTag)
+}
+
+export const WizgovWidgetClient = (props: WizgovWidgetClientProps) => {
+  const { "data-agency": dataAgency, scriptUrl } = props
```

Comment:
minor nit: can do the object destructuring within the method signature directly so the code is more straightforward